### PR TITLE
Update License Year

### PR DESCRIPTION
**Pull request by MrBrain295**
2020 to 2021

### CODE REVIEW

**Comment by SideFli** [comment]
ok. make sense to me

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Anurag Hazra
+Copyright (c) 2021 Anurag Hazra
```
Comment (mmvanheusden) [inline LICENSE:3]: Maybe change (c) to ©

**Review by mmvanheusden — CHANGES_REQUESTED** [review]
Maybe change (c) to ©

File: LICENSE (hunk 1 above)
Comment (MrBrain295) [reply LICENSE:3]: Ok

**Review by xiaopangju — APPROVED** [review]

**Pull request merged**